LottieSample/src/main/kotlin/com/airbnb/lottie/samples/TrimView.kt

```diff
@@ -15,14 +15,10 @@ class TrimView @JvmOverloads constructor(
 ) : FrameLayout(context, attrs, defStyleAttr) {
 
     private val leftAnchor by lazy {
-        val iv = ImageView(context)
-        iv.setImageResource(R.drawable.ic_trim)
-        iv
+        ImageView(context).apply { setImageResource(R.drawable.ic_trim) }
```

Comment:
make the code more concise.
reference: [http://kotlinlang.org/api/latest/jvm/stdlib/kotlin/apply.html](http://kotlinlang.org/api/latest/jvm/stdlib/kotlin/apply.html)